odoo_module_migrate/__main__.py

```diff
@@ -101,6 +101,12 @@ def get_parser():
         " the changes. (using git add and git commit command)"
     )
 
+    # TODO: Move to `argparse.BooleanOptionalAction` once in Python 3.9+
+    main_parser.add_argument(
+        "-npc", "--no-pre-commit", dest="pre_commit", action="store_false",
```

Comment:
I see the name and the dest not consistent. It should be --pre-commit (and -pc)

Don't you think ?
Otherwise lgtm. THanKs !

Reply:
Well, as commented in https://github.com/OCA/odoo-module-migrator/pull/55#discussion_r669384820, `--pre-commit` would be the default option and you would use `--no-pre-commit` to disable it, so it makes sense only to declare this last one.

When you use `store_false`, [`argparse` will set the destination variable as `True` by default and `False` if the argument is present](https://docs.python.org/3.6/library/argparse.html#action), which is what we need. I think this approach is clearer than using a negative boolean, like currently implemented in `--no-commit`. The practical effect is the same, it is just easier to understand codewise.

Reply:
My bad. I didnt know the store_false feature. Lgtm.